telegraf/init.sls

```diff
@@ -8,12 +8,19 @@
 {# There is data available so we think telegraf should be installed #}
 {% set role = salt['pillar.get']('netbox:role:name') %}
 
+influxdb-repo-key:
+  cmd.run:
+    - name: "curl https://repos.influxdata.com/influxdb.key | gpg --dearmor > /usr/share/keyrings/influxdb-keyring.gpg"
+    - creates: /usr/share/keyrings/influxdb-keyring.gpg
+
 influxdb-repo:
   pkgrepo.managed:
     - humanname: Jitsi Repo
     - name: deb https://repos.influxdata.com/{{ grains.lsb_distrib_id | lower }} {{ grains.oscodename }} stable
```

Comment:
Does this one need a `[signed-by=...]` option as well?

Reply:
you are right. changed it now